typings/index.d.ts

```diff
@@ -864,6 +867,46 @@ declare module "noblox.js" {
         awardingUniverse: BadgeUniverse
     }
 
+    //Inventory
+
+    interface AssetOwner
+    {
+        userId: number;
+        username: string;
+        buildersClubMembershipType: number;
+    }
+    
+    interface CollectibleEntry {
+        userAssetId: number;
+        serialNumber?: number;
+        assetId: number;
+        name: string;
+        recentAveragePrice: string;
+        originalPrice?: number;
+        assetStock?: number;
+        buildersClubMembershipType: number;
+    }
+
+    interface Collectibles {
+        collectibles: CollectibleEntry[];
+    }
+    //
+
+    interface InventoryEntry {
+        assetId: number;
+        name: string;
+        assetType: string;
+        created: string;
```

Comment:
Could we convert this to a Date?
Looks to be a standard ISO 8601 date string, so new Date(...) should be sufficient.

Reply:
Same goes for all other updated/created fields